### ISSUE TITLE
[💡 Feature Request]: Feature Request Every time assistant starts, you can add a command for it to execute, for example "every time Google Assistant start turn the table lights on"

### ISSUE THREAD

**Issue by Lucixs** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#760]
### What's the Problem?

Feature Request Every time assistant starts, you can add a command for it to execute, for example "every time Google Assistant start turn the table lights on"

### What's your solution or idea?

Feature Request Every time assistant starts, you can add a command for it to execute, for example "every time Google Assistant start turn the table lights on"

### Alternatives Considered

Feature Request Every time assistant starts, you can add a command for it to execute, for example "every time Google Assistant start turn the table lights on"

### App Version

v1.0.0-rc.2

### OS Version

Windows 11

### Pre-submission Checklist

- [X] I have checked [the issue tracker](https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/issues?q=is%3Aissue+label%3A%22%E2%9C%A8+enhancement%22+) and didn't find any duplicate issue
- [X] This issue only covers one particular feature request

**Comment by imclaux** [comment]
I'd like this a lot.

**Comment by im-coder-lg** [comment]
Yeah but this could be counterintuitive, since if it is set on every start, you can't find required data quickly.

**Comment by imclaux** [comment]
Could be set to do that after 30s, not instantly. But it would be pretty neat to have the lights turn on automatically after the pc boots up.

**Comment by im-coder-lg** [comment]
Hmm, then it has to be a startup job, huh? It could be possible, but we need to code, and I don't have much experience with JS, let's wait and see if this is a must-want.
